src/pages/GenerateKeys/Option2.tsx

```diff
@@ -282,8 +282,9 @@ export const Option2 = ({
             --chain {ETH2_NETWORK_NAME.toLowerCase()}
           </span>{' '}
           for {ETH2_NETWORK_NAME.charAt(0).toUpperCase()}
-          {ETH2_NETWORK_NAME.toLowerCase().slice(1)} testnet, otherwise the
-          deposit will be invalid.
+          {ETH2_NETWORK_NAME.toLowerCase().slice(1)}
+          {ETH2_NETWORK_NAME.toLowerCase() !== 'mainnet' && ' testnet'},
```

Comment:
I think it is better to use the `IS_MAINNET` environment variable here than checking equality on the `ETH2_NETWORK_NAME` one

Reply:
Right, that would be better.

Reply:
addressed in #252